File: AppMusic/app/src/main/java/com/example/appmusic/ui/base/BaseFragment.java
package com.example.appmusic.ui.base;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import dagger.hilt.android.AndroidEntryPoint;

@AndroidEntryPoint
public class BaseFragment extends Fragment {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }
}

File: AppMusic/app/src/main/java/com/example/appmusic/ui/base/BaseFragment.kt
package com.example.appmusic.ui.base

import android.os.Bundleimport

androidx.fragment.app.Fragmentimport dagger.hilt.android.AndroidEntryPoint
@AndroidEntryPoint
open class BaseFragment : Fragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }
}